infra/cray_infra/util/default_config.py

```diff
@@ -32,7 +32,8 @@ class Config(BaseModel):
 
     gpu_memory_utilization: float = 0.6
     max_model_length: int = 32768
-    dtype: str = "half"
+    dtype: str = "bfloat16"
+    #TODO: how to detect gpu type? device: str = "cuda"
```

Comment:
if we could specify cuda vs. not -> then that can be loaded in config during create_vllm. But I am unsure about the appropriate place for that call is.